src/sentry/users/api/serializers/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .identity import *  # noqa: F401,F403
+from .identityprovider import *  # noqa: F401,F403
+from .useremail import *  # noqa: F401,F403
+from .userip import *  # noqa: F401,F403
+from .userrole import *  # noqa: F401,F403
```

Comment:
If we don't need this barrel file to satisfy import side-effects we could skip it.

Reply:
Yeah, currently I think it's needed so that all the serializers will be registered, but we can just do explicit serializers in the endpoints instead to knock 2 birds with 1 stone